specs/VersionInfo/VersionInfo.md

```diff
@@ -0,0 +1,188 @@
+                      $"Minimum Windows App Runtime version ${build_version.Major}.${build_version.Minor}.${build_version.Build}.${build_version.Revision}\n"
+                      "\n" +
+                      $"Running with Windows App SDK ${release.Major}.${release.Minor}.${release.Patch}${release.FormattedVersionTag}\n" +
+                      $"Windows App Runtime version ${runtime.version.Major}.${runtime.version.Minor}.${runtime.version.Build}.${runtime.version.Revision}";
+        MessageBox.Show(message, "About MyApp", MessageBoxButtons.OK, MessageBoxIcon.Information);
+    }
+}
+```
+
+# Remarks
+
+Version information is compiled into `Microsoft.WindowsAppRuntime.Insights.Resource.dll` as the
+product is assembled (i.e. when the version is defined). The VersionInfo API retrieves this
+information via an internal API.
+
+# 6. API Details
+
+```c# (but really MIDL3)
+namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime.VersionInfo
+{
+    [contractversion(1)]
+    apicontract VersionInfoContract{};
+
+    /// Version information for the Windows App SDK release.
+    [contract(VersionInfoContract, 1)]
+    runtimeclass ReleaseInfo : Windows.Foundation.IStringable
+    {
+        ReleaseInfo();
+
+        /// Return the current release information.
+        static ReleaseInfo Current{ get; };
+
+        /// The major version of the Windows App SDK release.
+        UInt16 Major{ get; };
+
+        /// The minor version of the Windows App SDK release.
+        UInt16 Minor{ get; };
+
+        /// The patch version of the Windows App SDK release.
+        UInt16 Patch{ get; };
+
+        /// The major and minor version of the Windows App SDK release, encoded as a 32-bit integer (0xMMMMNNNN where M=major, N=minor).
+        UInt32 MajorMinor{ get; };
+
+        /// The Windows App SDK release's channel; for example, "preview", or empty string for stable.
+        String Channel{ get; };
+
+        /// The Windows App SDK release's version tag; for example, "preview2", or empty string for stable.
+        String VersionTag{ get; };
+
+        /// The Windows App SDK release's short-form version tag; for example, "p2", or empty string for stable.
+        String VersionShortTag{ get; };
+
+        /// The Windows App SDK release's version tag, formatted for concatenation when constructing identifiers; for example, "-preview2", or empty string for stable.
+        String FormattedVersionTag{ get; };
+
+        /// The Windows App SDK release's short-form version tag, formatted for concatenation when constructing identifiers; for example, "-p2", or empty string for stable.
+        String FormattedVersionShortTag{ get; };
+    };
+
+    /// Version information for the Windows App SDK runtime.
+    [contract(VersionInfoContract, 1)]
+    runtimeclass RuntimeInfo : Windows.Foundation.IStringable
```

Comment:
Not sure this needs to be an RTC on its own. Feels like all of these are static values.

Reply:
DISCUSSION: Yes. See above